### DISCUSSION TITLE
Code Review IM-11

### DISCUSSION THREAD

**Discussion by YevheniiSemyvolos** [HowProgrammingWorks/LiveQA#61] [Code review]
1-й курс 2-й семестр. Розробив браузерну гру "Let's throw dice!"
Буду вдячний за ревью [кода](https://github.com/EugeneSemivolos/LetsThrowDice/tree/dev)

**Comment by tshemsedinov** [comment]
https://github.com/EugeneSemivolos/LetsThrowDice/issues/4
